src/_help/publish-documentation/documentation-access-management.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Documentation access management
+---
+
+- TOC
+{:toc}
+
+## Public or Private?
+
+Bump.sh allows you to choose whether a documentation is public or private.
+
+By "public," we mean documentation that is accessible to any visitor who has access to it, either by finding it through a direct link or through a search engine.
+Public documentation is indexed by search engines and can be found by anyone without any restrictions. It is not possible to restrict access to any part of your documentation; it will be fully accessible.
+
+Private documentation, on the other hand, is only accessible to users who have been individually invited. There are two types of invited users: those who are part of the same organization and "external" guests.
+When documentation is set as private, it is completely private. It is excluded from indexing by search engines, and only authenticated users who know its URL can access it.
+
+![](/images/help/documentation-access-choice.png)
+
+## Private
+
+### Invite external guests
+
+To invite people to access a private documentation, click on the `Share Access` button, which is visible in the settings of that documentation.
+
+![](/images/help/share-access-button.png)
+
+From there, you just need to enter the email address of the invited person.
+
+![](/images/help/share-access.png)
+
+They will receive an email inviting them to sign up on Bump.sh. This step is essential to ensure the security of access to your documentation.
```

Comment:
Should we add a note about the fact that this isn't possible (yet) on hubs?

Reply:
Not 100% sure, I think that should be in Hubs section instead?